api/src/main/java/io/minio/ListIncompleteUploadsBucketArgs.java

```diff
@@ -0,0 +1,39 @@
+package io.minio;
+
+/** Argument class of @see #listIncompleteUploads(ListIncompleteUploadsBucketArgs args). */
+public class ListIncompleteUploadsBucketArgs extends BucketArgs {
```

Comment:
We would need to sync name with API i.e. `ListIncompleteUploadsArgs` is more appropriate.